pyincore/models/fragilitycurve.py

```diff
@@ -101,31 +101,21 @@ def calculate_limit_state_probability(self, hazard_values: dict, fragility_curve
 
         return probability
 
-    def get_building_period(self, num_stories):
-        """Get building period from the fragility curve.
-        Args:
-            num_stories (int): Number of building stories.
-        Returns:
-            float: Building period.
-        """
+    def get_building_period(self, fragility_curve_parameters, **kwargs):
```

Comment:
Can you add these docstings here:

def get_building_period(self, fragility_curve_parameters, **kwargs):
        """ Get building period from the fragility curve.
        
        Args:
            fragility_curve_parameters (dict): Fragility curve parameters.
            **kwargs: Keyword arguments.
        
        Returns:
            float: Building period.
            
        """